README.md

```diff
@@ -67,9 +67,9 @@ class PaymentService{
 ### Register corda node
 
 ```java
-class Main{
-    public static void main(){
-        PaymentService service = new PaymentSerice();
+class Main {
+    public static void main() {
```

Comment:
Since this code might and probably would be copy-pasted, it is better to have it in the ready-to-run state. As such, please, format main methods with the standard `public static void main(String[] args) {`